lib/templates.js

```diff
@@ -191,30 +191,32 @@ var templateSpecs = {
           console.log('LESS Generation Error', e);
           return;
         }
-        // With scss enabled this should be appendFileSync
-        fs.writeFileSync(strategy.output,
```

Comment:
Not sure why, but the less builder runs after scss, so if the less does `writeFileSync` it will overwrite the contents of the build file, so `appendFileSync` needs to be used. If you look in `build-shared/juju-ui/assets/juju-gui.css` you can see that the scss content will be at the top of the file.